app/javascript/components/components/Header.js

```diff
@@ -31,9 +31,76 @@ class Header extends Component {
               <a href={new_user_route} className="nav-link">Sign Up</a>
             </NavItem>
           }
+          {!logged_in &&
+            <NavItem>
+              <a href="/apartmentindex" className="nav-link">All Apartments</a>
```

Comment:
All internal navigation (meaning to other React component) should use a `NavLink to=` from react-router-dom. Any external navigation (meaning to Devise pages) should be `a href` tags.